[WC-1337]: Fix barcode-scanner (#35)
mendix/web-widgets: packages/pluggableWidgets/barcode-scanner-web/CHANGELOG.md

## [Unreleased]

### Fixed

> -   We fixed issue with camera, when it keep capturing video after widget is closed.

Reviewer: ```suggestion
-   We fixed an issue where the camera kept capturing after the widget was closed.
```
